tests/mocks/malga-tests-mocks.ts

```diff
@@ -0,0 +1,46 @@
+import { MalgaFormElements } from '../../src/common/interfaces/form'
+
+export const malgaConfigurations = (isSandbox: boolean) => {
+  return {
+    apiKey: '17a64c8f-a387-4682-bdd8-d280493715e0',
+    clientId: 'd1d2b51a-0446-432a-b055-034518c2660e',
+    options: {
+      sandbox: isSandbox,
+    },
+  }
+}
+
+export const configureFormSubmissionMock = (eventSubmit?: any) => {
+  eventSubmit
+    ? (window.HTMLFormElement.prototype.submit = eventSubmit)
+    : (window.HTMLFormElement.prototype.submit = () => {})
+  const onSubmit = vi.fn()
+  onSubmit.mockImplementation((event) => {
+    event.preventDefault()
+  })
+}
```

Comment:
Por padrão mantemos funções com `function` e callbacks com `arrow function`

```suggestion
export function malgaConfigurations(isSandbox: boolean) {
  return {
    apiKey: '17a64c8f-a387-4682-bdd8-d280493715e0',
    clientId: 'd1d2b51a-0446-432a-b055-034518c2660e',
    options: {
      sandbox: isSandbox,
    },
  }
}

export function configureFormSubmissionMock(eventSubmit?: any) {
  eventSubmit
    ? (window.HTMLFormElement.prototype.submit = eventSubmit)
    : (window.HTMLFormElement.prototype.submit = () => {})
  const onSubmit = vi.fn()
  onSubmit.mockImplementation((event) => {
    event.preventDefault()
  })
}
```